src/vm/gcenv.ee.cpp

```diff
@@ -371,7 +371,7 @@ bool GCToEEInterface::IsPreemptiveGCDisabled()
     Thread* pThread = ::GetThread();
     if (pThread)
```

Comment:
`return (pThread && pThread->PreemptiveGCDisabled())` to match the style of other places.